### PR TITLE
saved trips are displaying on DOM

### PR DESCRIPTION
**Pull request by calebsjames**
# Description

Saved trips are now displaying on DOM. They still need to be moved to the side and be displayed in cards.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Testing Instructions
git fetch --all
git checkout cj-aside
serve

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings or errors
- [x] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by nathanRomad — DISMISSED** [review]
your css changes look good. I like the way the dropdown bars are setup.
The saveItinerary button did not render to the aside but I can see the aside..
looks like you made a ton of progress on this module!

File: scripts/itineraries/ItinerarySave.js [hunk 1]
```diff
@@ -97,19 +112,29 @@ eventHub.addEventListener("click", clickEvent => {
...
+        if (newItineraryObject.parkName !== "" && newItineraryObject.eateryName !== "" && newItineraryObject.bizarrieName !== "") {
+            debugger  
+            console.log(newItineraryObject.parkName)
+            // assign captured variables to new object
+            const newItinerary = {
+                "parkId": newItineraryObject.parkId ,
+                "parkName": newItineraryObject.parkName,
+                "eateryId": newItineraryObject.eateryId,
+                "eateryName": newItineraryObject.eateryName,
+                "bizarraryId": newItineraryObject.bizarrieId,
```
Comment (yasmeencole) [inline scripts/itineraries/ItinerarySave.js:124]: Bizarre is spelled incorrectly

**Review by kylieanyce — APPROVED** [review]
looks great!

**Review by yasmeencole — APPROVED** [review]
looks good

**Pull request merged**